api/eventsingress_api.md

```diff
@@ -0,0 +1,49 @@
+<!--:nav_order:2-->
+
+# Events Publishing API
+
+## Prerequisites
+
+1. Create a project and a workspace in the [`Adobe I/O Developer console`](https://www.adobe.io/apis/experienceplatform/console/docs.html#!AdobeDocs/adobeio-console/master/projects-empty.md)
```

Comment:
It's called the Adobe Developer Console (no, I/O, proper capitalization)
Please fix throughout

Reply:
done : see https://github.com/AdobeDocs/adobeio-events/pull/77/commits/50413486082fe3d4e1de1ed1039ff74271eb223b